onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4036,12 +4051,21 @@ def aten_norm_except_dim(v: TensorType, pow: int = 2, dim: int = 0) -> TensorTyp
     raise NotImplementedError()
 
 
+@torch_op("aten::normal")
 def aten_normal(
-    self: TensorType, mean: float = 0.0, std: float = 1.0, generator: Optional[str] = None
-) -> TensorType:
+    self: TTensor,
+    mean: float = 0.0,
+    std: float = 1.0,
+    generator: str = "none",  # pylint: disable=unused-argument
+) -> TTensor:
```

Comment:
Looks like the output type can only be float, according to the doc?

```suggestion
) -> FloatType:
```